### ISSUE TITLE
Consider adding support for non true color environments like 256 colors.

### ISSUE THREAD

**Issue by norcalli** [norcalli/nvim-colorizer.lua#8]
The tricky thing here is that what we would be showing wouldn't be accurate if we attempted to approximate true RGB codes which aren't exact matches to your colorscheme. Even the 216 color cube could be overridden, but that would be unlikely considering most people only modify the 16 colors.

Therefore, there are 2 approaches possible here:

- Try to approximate values with a setting like `approximate_true_colors` which makes it very clear what it's doing
- Only highlight values we know to be 100% correct.

I don't have a use-case for this, so I need feedback from the community to see what would be useful. Currently, would recommend disabling colorizer if termguicolors isn't set with

```vim
if &termguicolors
lua require'colorizer'.setup()
endif
```

**Comment by skamsie** [comment]
Yes please! I am using the `terminal.app` which does not have true color support and I would love to be able to use your plugin.

Between the options that you listed, I think this one is better

> Try to approximate values with a setting like approximate_true_colors which makes it very clear what it's doing

**Comment by g6ai** [comment]
Second!
